### PR TITLE
Chore: Validate size of VS Code extension bundle

### LINKED ISSUE #2955: Validate size of VS Code extension bundle
> Should we do something similar to the browser extension where we have a size budget?

Yes, I'll file an issue to track that.

_Originally posted by @antross in https://github.com/webhintio/hint/pull/2954#issuecomment-529633258_

This will help us ensure we don't add large dependencies unintentionally.

### PR DESCRIPTION
**Pull request by antross**
Fix #2955

## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [ ] Added/Updated related documentation.
- [ ] Added/Updated related tests.

### CODE REVIEW

File: packages/extension-vscode/package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     ],
     "timeout": "1m"
   },
+  "bundleSize": 75000,
```
Comment (molant) [inline packages/extension-vscode/package.json]: 😍

File: packages/extension-vscode/scripts/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (molant) [inline packages/extension-vscode/scripts/tsconfig.json]: Why do we need a new `tsconfig.json` under `scripts`? Is it for `checkJS`? If so, can't we just add it to the top of the file (or maybe add that option in the root of our project?)

Comment (antross) [reply packages/extension-vscode/scripts/tsconfig.json]: Yes, this is for `checkJS`. We've used this across a few of our packages now, so adding to the root of our project seems like the right approach going forward. I'll update the PR.

File: packages/extension-vscode/scripts/bundle-size.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+const packageJSON = require('../package.json');
+
+const filename = `${path.resolve(__dirname, '..')}/vscode-webhint-${packageJSON.version}.vsix`;
+
+fs.stat(filename, (err, stats) => {
+    if (err) {
+        throw new Error(`Reading bundle failed: ${err}`);
+    }
+
+    const sizeKB = Math.round(stats.size / 1000);
```
Comment (sarvaje) [inline packages/extension-vscode/scripts/bundle-size.js]: Nit: Shouldn't this be `const sizeKB = Math.round(stats.size / 1024);`?

Comment (antross) [reply packages/extension-vscode/scripts/bundle-size.js]: Either that or a lower-case "k" according to wikipedia. https://en.wikipedia.org/wiki/Kilobyte

Honestly doesn't matter much in this case as we're using this just to keep tabs on bundle size growth. I went for consistency with what we were already doing for the browser extension.

Comment (sarvaje) [reply packages/extension-vscode/scripts/bundle-size.js]: hehehe, then `1000` is ok :)

File: packages/extension-vscode/scripts/tsconfig.json (hunk 2 above)
Comment (molant) [reply packages/extension-vscode/scripts/tsconfig.json]: sgtm

**Comment by molant** [comment]
looks like there are some merging conflicts. Otherwise this should be ready to merge.

**Pull request merged**